src/styles/blocks/star.scss

```
@@ -0,0 +1,18 @@
+.card__stars {
+  display: flex;
+  &-star {
+    width: 16px;
+    height: 16px;
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+  &-star:not(:last-child) {
+    margin-right: 4px;
+  }
```

Code will be more readable if you will add spaces between nesting selectors
```suggestion
  }
  
  &-star:not(:last-child) {
    margin-right: 4px;
  }
```